qiskit/transpiler/passes/calibration/exceptions.py

```diff
@@ -14,7 +14,7 @@
 from qiskit.exceptions import QiskitError
 
 
-class CalibrationNotAvailable(QiskitError):
+class CalibrationNotAvailableError(QiskitError):
```

Comment:
Its docstring comments that it's being used as a signal, so I'd argue it shouldn't have an `Error` suffix (nor derive from `QiskitError`, but that's completely separate).

Reply:
Reverted.